website/docs/en/config/output/minify.mdx

```diff
@@ -38,7 +38,7 @@ export default {
 ```
 
 :::tip
-This approach is usually used for debugging and troubleshooting. It is not recommended to disable code minification in production builds, as it will significantly degrade page performance.
+This approach is used for debugging and troubleshooting. Disabling code minification in production builds significantly degrades page performance.
```

Comment:
```suggestion
This approach is used for debugging and troubleshooting. We do not recommend to disable code minification in production builds, as it will significantly degrade page performance.
```